src/app/features/comics/services/comics-marvel.service.ts

```diff
@@ -0,0 +1,49 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { SpinnerService } from '@core/services/spinner.service';
+import { map, Observable, tap } from 'rxjs';
+import { CustomMarvelComic, MarvelApiComicsResponse, MarvelComic } from '../models/comics.model';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
why do you need it `providedIn: 'root'`?

Reply:
agreed seems I don't need it in data service